requirements.txt

```diff
@@ -1,2 +1,3 @@
 flask
 requests
+configparser
```

Comment:
I think this is part of the [python standard library](https://docs.python.org/3/library/configparser.html), so you don't need to add this here. We should make sure that we are running python 3 while developing this as python 2 is getting deprecated at some point.